cargo/lib/dependabot/cargo/file_fetcher.rb

```diff
@@ -20,6 +20,21 @@ def self.required_files_message
         "Repo must contain a Cargo.toml."
       end
 
+      def package_manager_version
+        channel = if rust_toolchain
+                    TomlRB.parse(rust_toolchain.content).fetch("toolchain", nil)&.fetch("channel", nil)
+                  else
+                    "default"
+                  end
+
+        {
+          ecosystem: "cargo",
+          package_managers: {
+            "channel" => channel
```

Comment:
Thinking aloud: 
Looking at https://rust-lang.github.io/rustup/overrides.html#channel, it looks like this will result in values like `"nightly-2020-07-10"`... 

I wonder if we'll ever want to query by "nightly" vs "stable"? For example, the "nightly" will have a super long tail so difficult to group by it... 

I suppose we can just do whatever extraction / grouping in the data consumer later, and "nightly" w/o the associated date is often somewhat meaningless.

So probably good to just go with this for now and call it good.